coding.md

```diff
@@ -164,7 +187,14 @@ A space should be used after a keyword of the conditional instruction. Opening a
 
 ## Type definition
 
-New types can only be defined if it is absolutely necessary.
+New types can only be defined if it is absolutely necessary. if `typedef` is used for a structure/union, structure/union should be left anonymous if possible:
+
+```c
+    typedef struct {
+       int foo;
+       int bar;
+    } foobar_t;
```

Comment:
Well, probably wide discussion needed about _t suffix :)

Reply:
Right now it's a fact and we don't present any alternative, so I decided to stick with it